.github/workflows/build-docker-release.yml

```diff
@@ -88,10 +88,18 @@ jobs:
         with:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
+      - name: Set tag variable
+        id: vars
+        run: echo ::set-output name=tag::$(echo ${GITHUB_REF#refs/*/})    
       - run: |
           npm install --package-lock-only
           npm ci && npm cache clean --force
-          npm publish --access public
+          if [[ "${{ steps.vars.outputs.tag }}" == *beta* ]];
+            then
+              npm publish --tag beta
+            else
+              npm publish --access public
```

Comment:
I don't think we need `--access public` here - unscoped packages (no @something) are public by default.

Reply:
`--access public` removed from all `npm publish` places.